tools/airgap_linter.py

```diff
@@ -54,6 +54,9 @@ def is_bad_uri(uri, file_name):
         ".mesos/",
         "$MESOS_CONTAINER_IP",
         "${MESOS_CONTAINER_IP}",
+        "$LIBPROCESS_IP",
```

Comment:
Has this been ported upstream?

Reply:
?

Reply:
@porridge which upstream is meant specifically? if it is about `dcos-commons` then there's a PR https://github.com/mesosphere/dcos-commons/pull/2737